### PR TITLE
Work With Us - replacing bootstrap grid with flexbox

### PR DESCRIPTION
**Pull request by kimrass14**
https://trello.com/c/1GQbTB31

The bootstrap grid classes and what each were changed to:
 .col-md-10 --> .iconContainer 
 .col-md-3 --> .iconTile (applied when minimum width of 768px and displays 4 icons per row)
 .col-xxssm-6 --> .iconTile-sm (applied when minimum width of 361px and displays 2 icons per row)

I looked at the bootstrap classes css which uses flexbox, so applied that to the newly created classes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,39 +16,39 @@ <h1 class="text-center" id="workWithUs">Work with Us</h1>
       </p>
     </div>
     <div class="padded-content workwithusContent">
-      <div class="row workwithusIconsRow col-md-10">
-        <div class="col-md-3 col-xxssm-6">
+      <div class="row workwithusIconsRow iconContainer">
```
Comment (jrapala) [inline index.html:19]: Sorry if this wasn't clear, but the `row` class also comes from Bootstrap (I think we add some additional styles to it in `main.scss`). 
Can we update the `row` class to include the styles that Bootstrap gives us?

File: index.html [hunk 2]
```diff
@@ -16,39 +16,39 @@ <h1 class="text-center" id="workWithUs">Work with Us</h1>
...
+        <div class="iconTile iconTile-sm">
```
Comment (jrapala) [inline index.html:20]: It seems like every element that uses `iconTile` uses `iconTile-sm` as well. Can we group all of the styles into one class instead?

File: scss/main.scss [hunk 3]
```diff
@@ -701,6 +701,22 @@ h1,h2,h3 {
...
+  position: relative;
+  width: 100%;
+  min-height: 1px;
+  padding-right: 15px;
+  padding-left: 15px;
+}
+
+.iconContainer {
+  -webkit-box-flex: 0;
+    -ms-flex: 0 0 83.33333%;
```
Comment (jrapala) [inline scss/main.scss:715]: Nothing to change, but this reminds me that I should get an autoprefixer set up..
Won't need to do this in the future.

File: scss/workwithus.scss [hunk 4]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (jrapala) [inline scss/workwithus.scss:1]: This file may have been made by mistake? Can it be deleted?

**Review by jrapala — CHANGES_REQUESTED** [review]
Nice job! Left a few comments to clean this up a bit more.

File: index.html (hunk 1 above)
Comment (jrapala) [reply index.html:19]: Ignore this! I just noticed we have a separate ticket to handle the `row` class.

File: index.html (hunk 2 above)
Comment (kimrass14) [reply index.html]: combined into iconTile. Removed iconTile-sm

File: scss/main.scss [hunk 5]
```diff
@@ -1,1027 +1,1058 @@
...
+@import './custom/includes/card';
+@import './custom/includes/copy_to_clipboard';
+@import './custom/includes/navbar';
+@import './custom/includes/images';
+@import './custom/includes/jumbotron';
+@import './custom/includes/pagination';
+@import './custom/includes/social_links';
+@import './custom/includes/syntax_highlighting';
+@import './custom/includes/tables';
+@import './custom/includes/type';
```
Comment (jrapala) [inline scss/main.scss:15]: You seem to have some sort of formatter or linter turned on that turned all of the `"` to `'` and has changed the style of indentations. 
I agree that linters/auto-formatters are great, but using it here has changed the entire file. 
Is this a commit that you are able to revert easily? If not, I'm happy to help with that.

**Review by jrapala — CHANGES_REQUESTED** [review]
One comment re: formatting

**Pull request closed without merging**